### DISCUSSION TITLE
Possibility to add notes/hints inside ordered/unordered lists

### DISCUSSION THREAD

**Discussion by ahcsco** [GitbookIO/community#102] [Feature Requests]
### What's your feature/idea?

No possibility today to add a hint or note between two list items. The numbering is reset if you do this. 

### What's your desired outcome?

Add possibility to insert a hint or note after a list and have the numbering continue without getting reset.

### What's the impact of this for you?

- [ ] Critical
- [ ] High
- [X] Medium
- [ ] Low

**Comment by aikoii** [comment]
Hi @shimshim938!

This is already possible 🎉! You can now start the ordered list at a specified number and add other blocks in between.

Could you make sure you are running the latest version of our application, please? One way to do this is [a hard refresh](https://docs.gitbook.com/resources/faq/how-to-hard-refresh-your-browser).

**Reply by ahcsco** [reply]
Thanks, No it's not possible. Tried hard reset as well. 
There is no option of hint available between lists. List items also get reset if you insert para.
<img width="703" alt="Screenshot 2023-02-16 at 11 50 30" src="https://user-images.githubusercontent.com/112613597/219345126-d47f1a0d-356e-4797-aab8-6342ab297841.png">
<img width="766" alt="Screenshot 2023-02-16 at 11 50 49" src="https://user-images.githubusercontent.com/112613597/219345132-2e4b02fd-995e-4ece-945a-6dc71de7d676.png">

**Reply by aikoii** [reply]
Hi @shimshim938,

I'm so sorry for the late reply! This fell through cracks 🙏.

There’s definitely some work on this to do on our side to better support blocks between list items. For now, an ordered list _will_ get reset if you insert a block between them, as you already noticed. However, you are able to start a list at a specified number 🙂. 

Here is a short recording showing how it looks:

https://user-images.githubusercontent.com/106068994/221790604-0e4ac1c4-9c6f-4649-939c-3403b6dfd3fc.mp4
